src/main/java/skynet/runner/Capabilities.java

```diff
@@ -45,6 +46,25 @@ public static Serializable get(ExtensionContext extensionContext) throws Excepti
         }
     }
 
+    public static Serializable get(ITestContext testContext) throws Exception {
```

Comment:
Although it's easier to just create a new method for this that does the same thing as another it's still not a good practice. I suggest you use Generics for these changes you have to do now:

```suggestion
    public static <T> Serializable get(T extensionContext) {
```
or even better

```suggestion
    public static <T extends ExtensionContext & ITestContext> Serializable get(T extensionContext) {
```
You'll also have to modify the methods used inside it such as: `setAndroidCaps()` to somethings similar: 

```suggestion
private static <T extends ExtensionContext & ITestContext> UiAutomator2Options setAndroidCaps(T extensionContext) {
```
You can even  change the return type of `get()` method and use Generics instead of Serializable, but I'll leave this up to you since it's not necessary.

Please take in consideration this comment applies to other methods that you are creating/duplicating inside this class